src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -197,6 +202,7 @@ endif()
 ################################
 # Umpire
 ################################
+set(UMPIRE_DIR ${GEOSX_TPL_DIR}/chai)
```

Comment:
should it be umpire?

Reply:
I observed (see e.g. in host-configs/tpls.cmake) that both UMPIRE_DIR AND CHAI_DIR are set to ${GEOSX_TPL_DIR}/chai

Reply:
Yes indeed
```
if(EXISTS ${GEOSX_TPL_DIR}/chai)
  set(UMPIRE_DIR ${GEOSX_TPL_DIR}/chai CACHE PATH "" FORCE)
  set(CHAI_DIR ${GEOSX_TPL_DIR}/chai CACHE PATH "" FORCE)
endif()
```
 I would bet this is a copy paste forgetfulness. Do you confirm @corbett5 ?

Reply:
@TotoGaz the way we build CHAI currently, it also builds Umpire for us and it is located in `CHAI_DIR`. When we build with Spack Umpire is created in a separate directory.